keyboards/eek/silk_down/keymaps/default/keymap.c

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2020 klackygears
```

Comment:
The keymaps folder can stay at the top level, that way you only need a single copy of them.

Reply:
Oh, well, that's smart, lol! I've made the update.